cmd/kubeadm/app/constants/constants.go

```diff
@@ -214,6 +214,9 @@ var (
 
 	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
 	MinimumKubeletVersion = version.MustParseSemantic("v1.8.0-beta.0")
+
+	// MinimumUnstableVersion specifies the earliest version of the upcoming major release
+	MinimumUnstableVersion = version.MustParseSemantic("v1.9.0-alpha.1")
```

Comment:
Changed the version for other constants in https://github.com/kubernetes/kubernetes/pull/53181